Bottom: 3018c24d7c3e63f510056b45fa0bad070684ef47
Top:    6a1ab41714504367cd1a8bd99aab42151ba63c7f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:03:32 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
